pkg/kots/lint.go

```diff
@@ -820,6 +824,35 @@ func lintFileHasValidYAML(file SpecFile) []LintExpression {
 	return lintExpressions
 }
 
+func lintRenderedFilesYAMLValidity(renderedFiles SpecFiles) []LintExpression {
+	var lintExpressions []LintExpression
+	for _, renderedFile := range renderedFiles {
+		var doc interface{}
+		err := yaml.Unmarshal([]byte(renderedFile.Content), &doc)
+		if err != nil {
+			lintErrMsg := err.Error()
+			errLine, err := util.TryGetLineNumberFromValue(err.Error())
+			if err == nil && errLine > -1 {
+				lines := strings.Split(renderedFile.Content, "\n")
+				if len(lines) > errLine {
+					errLineContent := lines[errLine-1]
+					lintErrMsg = strings.Replace(lintErrMsg, fmt.Sprintf("line %d", errLine), fmt.Sprintf("(%s)", errLineContent), 1)
```

Comment:
Could we change this to read with the error first and then the line with the invalid syntax? example:
```
yaml: found character that cannot start any token: alb.ingress.kubernetes.io/inbound-cidrs: ***HIDDEN***/0
```

Reply:
addressed in 
https://github.com/replicatedhq/kots-lint/pull/159/commits/5bedadc630ecbe1e385b0c5fe0da1dfea8f32051#diff-1d1c16d1fd801c249928ba51957dc01e3720d412f68cb52bbb3cf7f29912122bR838-R841